Assets/MixedRealityToolkit/Sharing/Scripts/SyncModel/SyncObject.cs

```diff
@@ -124,7 +119,7 @@ private void InitializePrimitives()
             // Scan the type of object this is a look for the SyncDataAttribute
             Type baseType = GetType();
 
-#if USE_WINRT
+#if WINDOWS_UWP
```

Comment:
Should we be using `UNITY_WSA` or `WINDOWS_UWP` throughout the project?

What's the difference?

Reply:
I was going to create another issue to standardise on our #IF statements throughout the project.  As we head towards a more multi-platform architecture, having single definitions for each state will be critical.

Reply:
#1667 created for sorting out later

Reply:
`UNITY_WSA` runs both in the compiled UWP app and in the editor. From my tests, `WINDOWS_UWP` is equivalent to our use of `UNITY_WSA && !UNITY_EDITOR`, running only in the compiled UWP app.